webapp/src/main.ts

```diff
@@ -40,6 +40,18 @@ if (environment.production) {
 
   Auth.configure({
     oauth: oauth,
+    identityPoolId: 'YOUR_IDENTITY_POOL_ID',
+    region: environment.awsRegion,
+    userPoolId: environment.cognitoUserPoolId,
+    userPoolWebClientId: environment.cognitoAppClientId,
+    authenticationFlowType: 'USER_SRP_AUTH',
+    identityProviders: {
+      'GitHub': {
+        'client_id': 'YOUR_GITHUB_CLIENT_ID',
+        'client_secret': 'YOUR_GITHUB_CLIENT_SECRET',
+        'authorize_scopes': 'read:user,user:email'
+      }
+    }
```

Comment:
I am not sure if this is needed since it is already done in `Amplify.configure(awsconfig);`. Correct me if I am wrong and this is doing different things.